test/Feature/InsertSpec.hs

```diff
@@ -508,13 +516,24 @@ spec actualPgVersion = do
           ""
           { matchStatus = 201 }
 
-  describe "Inserting into VIEWs" $
-    it "returns a location header" $
-      post "/compound_pk_view"
-          [json|{"k1":1,"k2":"test","extra":2}|]
-        `shouldRespondWith`
-          ""
-          { matchStatus  = 201
-          , matchHeaders = [ "Location" <:> "/compound_pk_view?k1=eq.1&k2=eq.test"
-                           , "Content-Range" <:> "*/*" ]
-          }
+  describe "Inserting into VIEWs" $ do
+    context "requesting no representation" $
+      it "succeeds with 201" $
+        post "/compound_pk_view"
+            [json|{"k1":1,"k2":"test","extra":2}|]
+          `shouldRespondWith`
+            ""
+            { matchStatus  = 201
+            , matchHeaders = [ "Content-Range" <:> "*/*" ]
```

Comment:
Ah I really like that `matchHeaders = [ matchHeaderAbsent hLocation ]` above.

Can we add that here, too?

Reply:
Sure, will do. Should I leave the "Content-Range" in the matchHeaders? or replace it entirely with the matchHeaderAbsent?

Reply:
I think we can remove that. Keeps the test-cases more focused on the important details.